work/service/coordinator.go

```diff
@@ -331,23 +324,10 @@ func (c *Coordinator) completeWork(completion *coordinatorProcessingCompletion)
 	}
 }
 
-func (c *Coordinator) startTimer() {
+func (c *Coordinator) tick() <-chan time.Time {
```

Comment:
Were the changes in this file due to a bug? If so, what were you seeing? (The new code doesn't work exactly like the old code. There may be a repeated call to `requestAndDispatchWork` occasionally.)